contracts/ControlledGateway.sol

```diff
@@ -7,11 +7,9 @@ import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
 /**
  * @title ControlledGateway
  * @notice Base Contract for both L1 and L2 LPT gateways. Provides AccessControl.
- * Gateways can be paused by the governor to stop outgoing token migrations
+ * Gateways can be paused by the admin to stop outgoing token migrations
  */
 contract ControlledGateway is AccessControl, Pausable {
```

Comment:
If only a single role is used, making this Ownable instead of AccessControl will help in gas savings.